### ISSUE TITLE
[Accessibility: ACC-272.1] Dialog cannot be dismissed using ‘ESC’ key.

### ISSUE THREAD

**Issue by gere97** [SAP/fundamental-ngx#12698]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.
Dialog box cannot be closed with ESC key. Keyboard is the prerequisite for support of assistive technologies to enable users with disabilities to read and operate software applications, hence there should be a possibly to close the UI control using keyboard.
#### Can you handle that on the application side
No
#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All
#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
https://sap.github.io/fundamental-ngx/#/platform/value-help-dialog
Dialog boxes from that page cannot be closed with the esc key. 
#### Please provide relevant source code (if applicable).

`<div role="dialog" tabindex="-1" aria-modal="false" fdkresize="" cdktrapfocus="" cdkdrag="" cdkdragboundary=".fd-dialog" fd-scrollbar="" class="cdk-drag fd-scrollbar fd-dialog__content fdp-value-help-dialog cdk-drag-disabled" ng-reflect-enabled="" ng-reflect-boundary-element=".fd-dialog" ng-reflect-resize-boundary="[object HTMLElement]" ng-reflect-disabled="true" style="overflow: auto; position: relative; width: 1080px; height: 98%; max-width: 92%;"><!--container--><div cdkdraghandle="" class="cdk-drag-handle" ng-reflect-disabled="true"><fd-dialog-header dir="ltr"><div fd-bar="" class="fd-bar fd-bar--header fd-dialog__header is-compact" ng-reflect-fd-cozy="false" ng-reflect-bar-design="header"><h5 fd-title="" class="fd-title fd-title--h5 ng-star-inserted" id="fdp-vhd-header-1">`

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes
#### Did you search for similar issues?
Yes
#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@gere97 the fundamental ngx behvaior is aligned with ui5 and you can use keyboard to close the dialog by focusing on cancel button
